src/config.js

```diff
@@ -108,6 +108,9 @@ try {
 		BRANCH_PREFIX: getInput({
 			key: 'BRANCH_PREFIX',
 			default: 'repo-sync/SOURCE_REPO_NAME'
+		}),
+		FORK: getInput({
+			key: 'FORK'
```

Comment:
Please add a default value, in our case `false`.

You can also set the `FORK` option to `disableable` ([more info](https://github.com/BetaHuhn/action-input-parser#all-options)).

```suggestion
			key: 'FORK',
			default: false,
			disableable: true
```